AjaxFetch/js/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * get response from api through 'fetch'
+ * @param {string} path
+ */
+function getResponse(path) {
+  const root = 'https://jsonplaceholder.typicode.com';
+
+  fetch(`${root}/${path}`).then(
+    function(response){
+      response.json().then(
+        function(data){
+          localStorage.setItem(path, data);
+        });
+    });
+}
+
+
+window.addEventListener('load',() => {
+
+  window.values = {
```

Comment:
You can use this variable without window. Just define that on the top of document